### PR TITLE
KAA-1245 Shell style guide

### PR DESCRIPTION
**Pull request by forGGe**

### CODE REVIEW

**Comment by balldir** [comment]
It's tiny, but all described is reasonable. LGTM

**Comment by vadimol** [comment]
+

File: doc/Customization-guide/Code-style/Shell/index.md [hunk 1]
```diff
@@ -4,4 +4,25 @@ title: Shell
 permalink: /:path/
 sort_idx: 50
 ---
-Specific per-language code style guidelines. Check style configuration files for Eclipse and IDEA. Style check also must be automated in the build process (after code styles are defined here).
+
+- Prefer plain Bourne shell scripts when possible. Avoid using GNU Bash, zsh or fish since they are possibly will be missing in a particular environment.
```
Comment (rasendubi) [inline doc/Customization-guide/Code-style/Shell/index.md]: Add `(sh)` after `Bourne shell`. Most people don't know, how it's called and may confuse it with bash

Comment (rasendubi) [inline doc/Customization-guide/Code-style/Shell/index.md]: You can drop fish but include `ksh` or `csh`.

(minor) Add comma before `or`

Comment (rasendubi) [inline doc/Customization-guide/Code-style/Shell/index.md]: they are possibly will be missing -> they might be missing

File: doc/Customization-guide/Code-style/Shell/index.md [hunk 2]
```diff
@@ -4,4 +4,25 @@ title: Shell
...
+- Use [sha-bang](https://en.wikipedia.org/wiki/Shebang_(Unix)) at the beginning of each shell script.
```
Comment (rasendubi) [inline doc/Customization-guide/Code-style/Shell/index.md]: s/sha-bang/shebang

Comment (rasendubi) [inline doc/Customization-guide/Code-style/Shell/index.md]: s/beginning/first line

**Comment by vadimol** [comment]
All comments were addressed. 
Looks good for me.

**Comment by rasendubi** [comment]
LGTM

**Pull request merged**